static/src/stylesheets/module/crosswords/_clues.scss

```diff
@@ -35,3 +35,12 @@
 .crossword__clue--answered {
     color: colour(neutral-2);
 }
+
+.crossword__clue--display-group-order {
+
+    padding-left: 0px;
+
+    &:before {
+        position: relative;
```

Comment:
`position: static` is the initial value, or you can just do `position: initial`, if I remember correctly…

I like to leave a comment when I am "undoing CSS", so other people will know.